src/library-authoring/common/OrganizationDropdown.jsx

```diff
@@ -192,7 +204,7 @@ class OrganizationDropdown extends React.Component {
           handleFocus={this.handleFocus}
         />
         <div className="dropdown-container">
-          { this.state.dropDownItems.length > 0 ? this.state.dropDownItems : null }
+          { this.state.dropDownItems.length > 0 ? this.state.dropDownItems : dropDownEmptyList }
```

Comment:
Why we can't just place `noOptionsMessage` straight away?
```suggestion
          { this.state.dropDownItems.length > 0 ? this.state.dropDownItems : noOptionsMessage }
```

Reply:
because we need to return a message or null based on the focus state
`this.state.dropDownItems && this.state.isFocused ? noOptionsMessage : null`. Otherwise, the message will always be shown